.github/workflows/scripts/linux/build-dependencies-qt.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+
+set -e
+
+INSTALLDIR="$HOME/deps"
+NPROCS="$(getconf _NPROCESSORS_ONLN)"
+SDL=SDL2-2.0.22
+QT=6.3.0
+
+mkdir -p deps-build
+cd deps-build
+
+cat > SHASUMS <<EOF
+fe7cbf3127882e3fc7259a75a0cb585620272c51745d3852ab9dd87960697f2e  $SDL.tar.gz
+b865aae43357f792b3b0a162899d9bf6a1393a55c4e5e4ede5316b157b1a0f99  qtbase-everywhere-src-$QT.tar.xz
+3164504d7e3f640439308235739b112605ab5fc9cc517ca0b28f9fb93a8db0e3  qtsvg-everywhere-src-$QT.tar.xz
+fce94688ea925782a2879347584991f854630daadba6c52aed6d93e33cd0b19c  qttools-everywhere-src-$QT.tar.xz
+e7b567f6e43ffc5918d4aa825ce1eced66a00cb0a87133b2912ba5c1b2a02190  qtwayland-everywhere-src-$QT.tar.xz
+EOF
+
+curl -L \
+	-O "https://libsdl.org/release/$SDL.tar.gz" \
+	-O "https://download.qt.io/official_releases/qt/${QT%.*}/$QT/submodules/qtbase-everywhere-src-$QT.tar.xz" \
+	-O "https://download.qt.io/official_releases/qt/${QT%.*}/$QT/submodules/qtsvg-everywhere-src-$QT.tar.xz" \
+	-O "https://download.qt.io/official_releases/qt/${QT%.*}/$QT/submodules/qttools-everywhere-src-$QT.tar.xz" \
+	-O "https://download.qt.io/official_releases/qt/${QT%.*}/$QT/submodules/qttranslations-everywhere-src-$QT.tar.xz" \
```

Comment:
Not hash checking qttranslations. echo "Installing Qt Translations" is outside the "if" statement

Reply:
The ifs can probably go now tbh, I just had them in there when refactoring the script to make it run faster locally